styleguide/derek/_partials/_rug.ejs

```diff
@@ -1,12 +1,47 @@
 <!--[rug]-->
-<div class="container rug">
+<div class="container-fluid bg-blue-gray half-padding">
+<div class="container hidden-xs hidden-sm">
   <div class="row">
     <div class="col-sm-3">
-      <div class="col-sm-3">
-        <h5>We've got your back</h5>
-        <h4 class="text34">24x7x365</h4>
+      <h5 class="rug-subhead-medium no-margin">We've got your back</h5>
+      <h4 class="rug-subhead-large no-margin">24<small class="white">x</small>7<small class="white">x</small>365</h4>
```

Comment:
Yeah I wouldn't bother translating it, numbering like that is pretty much universal ~ plus it's kinda like a slogan, and they aren't usually translated either.